microsoft-365/admin/get-help-with-domains/remove-a-domain.md

```diff
@@ -160,7 +160,7 @@ To change your NS records, see [Change nameservers to set up Microsoft 365 with
 
 ## How long does it take for a domain to be removed?
 
-It can take as little as 5 minutes for Microsoft 365 to remove a domain if it's not referenced in a lot of places such as security groups, distribution lists, users, and Microsoft 365 groups. If there are many references that use the domain it can take several hours (a day) for the domain to be removed.
+It can take as little as 5 minutes for Microsoft 365 to remove a domain if it's not referenced in a lot of places such as security groups, distribution lists, users, aliases, shared mailboxes, resource mailboxes and Microsoft 365 groups. If there are many references that use the domain it can take several hours (a day) for the domain to be removed.
```

Comment:
```suggestion
It can take as little as five minutes for Microsoft 365 to remove a domain if it's not referenced in a lot of places such as security groups, distribution lists, users, aliases, shared mailboxes, resource mailboxes, and Microsoft 365 groups. If there are many references that use the domain it can take several hours (a day) for the domain to be removed.
```